plexus-java/src/main/java/org/codehaus/plexus/languages/java/jpms/LocationManager.java

```diff
@@ -401,18 +403,21 @@ private void selectRequires( JavaModuleDescriptor module,
                                  Map<String, Set<String>> availableProviders,
                                  Set<String> namedModules,
                                  boolean includeStatic, 
-                                 boolean includeTransitive )
```

Comment:
I think we should rename things a bit to prevent more confusion:
The first boolean-parameter looks more like `isRootModule`.
That way the last parameter, which is only used for the includeStatic, can be the actual includeParameter boolean.